.gitlab/package_build/build_agent_dmg.sh

```diff
@@ -11,11 +11,14 @@ fi
 # --- Setup environment ---
 unset OMNIBUS_GIT_CACHE_DIR
 unset OMNIBUS_BASE_DIR
-export INSTALL_DIR="$PWD/datadog-agent-build/bin"
-export CONFIG_DIR="$PWD/datadog-agent-build/config"
-export OMNIBUS_DIR="$PWD/omnibus_build"
+WORKDIR="/tmp"
+export INSTALL_DIR="$WORKDIR/datadog-agent-build/bin"
+export CONFIG_DIR="$WORKDIR/datadog-agent-build/config"
+export OMNIBUS_DIR="$WORKDIR/omnibus_build"
 export OMNIBUS_PACKAGE_DIR="$PWD"/omnibus/pkg
 
+sudo rm -rf "$WORKDIR" && mkdir -p "$INSTALL_DIR" "$CONFIG_DIR" "$OMNIBUS_DIR"
```

Comment:
are we sure there can't be anything else needed by someone else in /tmp for whatever reasons?